signalfx-metrics/src/main/java/com/signalfx/metrics/connection/HttpClientConnectionManagerFactory.java

```diff
@@ -39,7 +41,7 @@ private static class SSLConnectionSocketFactoryWithTimeout extends SSLConnection
     private final int timeoutMs;
 
     public SSLConnectionSocketFactoryWithTimeout(int timeoutMs) {
-      super(SSLContexts.createDefault(), SSLConnectionSocketFactory.STRICT_HOSTNAME_VERIFIER);
```

Comment:
I couldn't find an equivalent of this in httpclient5. Could use a recommendation.